contracts/TreasuryActions.cdc

```diff
@@ -78,7 +78,7 @@ pub contract TreasuryActionsV2 {
       )
     }
 
-    init(recipientVault: Capability<&{FungibleToken.Receiver}>, amount: UFix64, proposer: Address) {
+    init(recipientVault: Capability<&{FungibleToken.Receiver}>, amount: UFix64, proposer: Address, coinType: String) {
```

Comment:
i dont think we should add `coinType` as an arg here, since its just metadata that _could_ be set to whatever someone wanted.  I think its better to keep the metadata hardcoded on the FE UI or integrate some 3rd party API to fetch metadata clientside